src/SupportMessage/SupportMessage.tsx

```diff
@@ -0,0 +1,100 @@
+import React, { FC, ReactNode } from 'react'
+import styled, { css } from 'styled-components'
+
+import { Box } from '../Box'
+import { Icon } from '../Icon'
+import { theme } from '../theme'
+
+type BorderProps =
+  | { hasBorder?: false; borderColor?: never }
+  | { hasBorder: true; borderColor: string }
+
+export type SupportMessageProps = {
+  className?: string
+  children: ReactNode
```

Comment:
Based on the current design shouldn't this be just a string? Having it as a ReactNode means we can render as a child what we want